src/index.js

```diff
@@ -16,7 +16,11 @@
 
 import * as Types from '@babel/types';
 import path from 'path';
-import ParserHelpers from 'webpack/lib/ParserHelpers';
+let ParserHelpers;
+try {
+  ParserHelpers = require('webpack/lib/javascript/JavascriptParserHelpers'); // Webpack 5
+} catch (e) {}
+ParserHelpers = ParserHelpers || require('webpack/lib/ParserHelpers'); // Webpack 4
```

Comment:
Great stuff! Just one super minor question about the code:
Any particular reason why it's not like this?

```suggestion
} catch (e) {
  ParserHelpers = require('webpack/lib/ParserHelpers'); // Webpack 4
}
```

Not a blocker by any means, though. Just curious.

Reply:
I literally just went in and copied from the upstream repo. Now that I look at this, it definitely seems...strange. 

Could it be that sometimes it wouldn't throw? Strange that it wouldn't, but you'd end up with `ParserHelpers` being `undefined` and a broken plugin.

Reply:
Yeah, I thought about it, too, but wasn't sure if this can actually happen. Anyway, let's play it safe then and just go with the upstream changes 👍